embedded-graphics/src/image/image_bmp.rs

```diff
@@ -231,6 +263,11 @@ mod tests {
         ))
         .unwrap()
         .translate(Point::new(-1, -1));
+
+        {
+            assert_eq!(image.into_iter().count(), 9);
+        }
+
```

Comment:
Hm?

Reply:
Leftover from an old attempt to fix some mutable reference issues, fixing